crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -1805,31 +1805,36 @@ pub(crate) fn convert_to_validators(
     validators: Vec<SuiValidatorSummary>,
     system_state: Option<NativeSuiSystemStateSummary>,
 ) -> Vec<Validator> {
-    let at_risk_validators: Option<BTreeMap<NativeSuiAddress, u64>> = system_state
-        .clone()
-        .map(|x| BTreeMap::from_iter(x.at_risk_validators));
-
-    let report_records: Option<BTreeMap<NativeSuiAddress, Vec<NativeSuiAddress>>> =
-        system_state.map(|x| BTreeMap::from_iter(x.validator_report_records));
+    let (at_risk, reports) = if let Some(NativeSuiSystemStateSummary {
+        at_risk_validators,
+        validator_report_records,
+        ..
+    }) = system_state
+    {
+        (
+            BTreeMap::from_iter(at_risk_validators),
+            BTreeMap::from_iter(validator_report_records),
+        )
+    } else {
+        Default::default()
```

Comment:
Not sure about this default. Could you explain the rationale behind returning default instead of None? Does default return None?

Reply:
It would create a tuple of empty BTreeMaps, instead of (None, None)

easier to work with empty BTreeMap than Option<BTreeMap>, latter would require more unwraps or Some checks

Reply:
I see, so it infers that the return type is BTreeMap so calling default will return an empty BTreeMap, got it. Thank you!